src/cli/test_command.zig

```diff
@@ -200,6 +201,14 @@ pub const CommandLineReporter = struct {
         this.summary.fail += 1;
         this.summary.expectations += expectations;
         this.jest.tests.items(.status)[id] = TestRunner.Test.Status.fail;
+
+        if (this.jest.bail == this.summary.fail) {
```

Comment:
We should reuse the same code for printing the existing summary here so we don't have 2 different formats. I like the "bail out" message, so we can keep that if it's a bail.